docs/multi-echo.rst

```diff
@@ -0,0 +1,45 @@
+Multi-echo fMRI
+===============
+In multi-echo (ME) fMRI, data are acquired for multiple echo times, resulting in
+multiple time series for each voxel.
+
+The physics of multi-echo fMRI
+------------------------------
+
+Why use multi-echo?
+-------------------
+
+Resources
+---------
+
+Journal articles
+****************
+* A `review`_ on multi-echo fMRI and its applications
+
+.. _review: https://www.ncbi.nlm.nih.gov/pubmed/28363836
+
+Videos
+******
+* An `educational session from OHBM 2017`_ by Dr. Prantik Kundu about multi-echo denoising
+* A `series of lectures from the OHBM 2017 multi-echo session`_ on multiple facets of multi-echo data analysis
+
+.. _educational session from OHBM 2017: https://www.pathlms.com/ohbm/courses/5158/sections/7788/video_presentations/75977
+.. _series of lectures from the OHBM 2017 multi-echo session: https://www.pathlms.com/ohbm/courses/5158/sections/7822
+
+Sequences
+*********
+* Multi-echo sequences: who has them and how to get them.
+
+Datasets
+********
+A small number of multi-echo datasets have been made public so far. This list is
+not necessarily up-to-date, so please check out OpenNeuro to potentially
+find more.
+
+* `Multi-echo fMRI replication sample of autobiographical memory, prospection and theory of mind reasoning tasks`_
+* `Multi-echo Cambridge`_
+* `Multiband multi-echo imaging of simultaneous oxygenation and flow timeseries for resting state connectivity`_
+
+.. _Multi-echo fMRI replication sample of autobiographical memory, prospection and theory of mind reasoning tasks: https://openneuro.org/datasets/ds000210/
```

Comment:
Is this rendering correctly ? it looks commented out on GitHub, but I'm not sure why.